fundingmanager.go

```diff
@@ -2337,12 +2354,8 @@ func (f *fundingManager) annAfterSixConfs(completeChan *channeldb.OpenChannel,
 		fndgLog.Infof("Announcing ChannelPoint(%v), short_chan_id=%v",
 			&fundingPoint, shortChanID)
 
-		// We'll obtain the min HTLC value we can forward in our
-		// direction, as we'll use this value within our ChannelUpdate.
-		// This constraint is originally set by the remote node, as it
-		// will be the one that will need to determine the smallest
-		// HTLC it deems economically relevant.
-		fwdMinHTLC := completeChan.LocalChanCfg.MinHTLC
+		// Determine min htlc amount that we can/want to forward.
+		fwdMinHTLC := f.getMinHtlcPolicy(&completeChan.LocalChanCfg)
```

Comment:
Nit: not caused by your change, but doesn't seem like we'd even need this parameter here since it's only for the channel update, which is not required to announce a channel. We are reusing `newChanAnnouncement` in `announceChannel` though.

Reply:
Added a commit to improve it a little. I don't want to start refactoring further in this pr.